### DISCUSSION TITLE
TP1 Macowins - Code Smells

### DISCUSSION THREAD

**Discussion by loyolacamil** [dds-jv/foro#348] [Consultas generales] [answered]
Buenas, me esta tirando los sgtes errores cuando hago el mvn verify.
<img width="562" alt="image" src="https://user-images.githubusercontent.com/43614089/164482877-ee4c968f-d009-441b-adca-985c26e5d8b8.png">
Entiendo que viene por el lado que no le gusta mucho el uso del Objeto Date, pero no se si dejarlo asi, o como solucionarlo.

**Comment by gastonprieto** [comment, marked as answer]
Buenas @loyolacamil !

no estoy muy seguro si el error es exactamente sobre eso, es posible. -cuando tengas un error pega los mensajes como texto formateado, asi podemos copiar-pegar en google o para que un futuro compañero encuentre tu problema-

Igual no deberias usar Date, fijate en [sobre los tps](https://dds-jv.github.io/pautas/sobre-los-trabajos-practicos) que dice que no hay que usar mas Date que es la API vieja de fechas de Java
